pyproject.toml

```diff
@@ -73,25 +74,25 @@ license = {text = "MIT"}
 requires-python = ">=3.10,<3.13"
 
 dependencies = [
-    "numpy~=1.25",
-    "phoenix6~=24.2.0",
+    "numpy~=2.1",
+    "phoenix6~=25.0.0b2",
     # robotpy[apriltag] pins numpy to a version that doesn't build cleanly on Python 3.12.
```

Comment:
This comment is no longer true.

- https://github.com/robotpy/robotpy-meta/blob/6aa03b1f4800f070c1c05ad3d9eddf6b4754ad35/pyproject.toml#L140-L141
- https://pypi.org/project/numpy/2.1.2/#files

```suggestion
```

But maybe you should consider using `robotpy[apriltag]` in this list now that's fixed.